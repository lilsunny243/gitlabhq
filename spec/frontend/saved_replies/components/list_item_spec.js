import { shallowMount } from '@vue/test-utils';
import ListItem from '~/saved_replies/components/list_item.vue';

let wrapper;

function createComponent(propsData = {}) {
  return shallowMount(ListItem, {
    propsData,
  });
}

describe('Saved replies list item component', () => {
  afterEach(() => {
    wrapper.destroy();
  });

  it('renders list item', async () => {
    wrapper = createComponent({ reply: { name: 'test', content: '/assign_reviewer' } });

    expect(wrapper.element).toMatchSnapshot();
  });
});
