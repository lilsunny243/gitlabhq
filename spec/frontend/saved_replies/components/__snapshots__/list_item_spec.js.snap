// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Saved replies list item component renders list item 1`] = `
<li
  class="gl-mb-5"
>
  <div
    class="gl-display-flex gl-align-items-center"
  >
    <strong>
      test
    </strong>
  </div>
   
  <div
    class="gl-mt-3 gl-font-monospace"
  >
    /assign_reviewer
  </div>
</li>
`;
